### PR TITLE
Oomox Theme Designer

### PR DESCRIPTION
**Pull request by cycool29**
Works pretty well on Raspberry Pi.

### CODE REVIEW

File: apps/Oomox Theme Designer/description [hunk 1]
```diff
@@ -0,0 +1,8 @@
+A tool to customize icons, xrdb and GTK themes. 
+
+Graphical application for generating different color variations of Oomox (Numix-based) and Materia (ex-Flat-Plat) themes (GTK2, GTK3, Cinnamon, GNOME, Openbox, Xfwm), Archdroid, Gnome-Color, Numix, Papirus and Suru++ icon themes. 
+Have a hack for HiDPI in gtk2.
```
Comment (Itai-Nelken) [inline apps/Oomox Theme Designer/description:4]: I'm not sure 
```suggestion
Has a hack for HiDPI in gtk2.
```

Comment (cycool29) [reply apps/Oomox Theme Designer/description:4]: I just copy from its [Github](https://github.com/themix-project/oomox#themix-gui-designer).

Comment (Itai-Nelken) [reply apps/Oomox Theme Designer/description:4]: what do you say @Botspot?

Comment (Botspot) [reply apps/Oomox Theme Designer/description:4]: I'll remove that line altogether - Raspberry Pi has its own way for handling high-res screens - pixel doubling.

**Comment by cycool29** [comment]
Anyone got this small window when launching the app?

image

**Comment by Botspot** [comment]
> Anyone got this small window when launching the app?
> 
> image

No, it launched fine for me. Could you try capturing the terminal output?

**Comment by cycool29** [comment]
> > Anyone got this small window when launching the app?
> > image
> 
> No, it launched fine for me. Could you try capturing the terminal output?

Now it worked for me. I missed the `--recursive` flag in git clone when testing.

**Comment by cycool29** [comment]
@Botspot I think this PR is ready to merge.

**Comment by Crilum** [comment]
Do we want to use `install_packages` and `purge_packages` instead of `pkg-install`?

**Comment by Botspot** [comment]
> Do we want to use `install_packages` and `purge_packages` instead of `pkg-install`?

Thanks for noticing. https://github.com/Botspot/pi-apps/commit/fe512aad82ae0643b56873886e65c413ddfdb73b

**Pull request merged**